pkg/virt-api/rest/subresource.go

```diff
@@ -212,60 +212,31 @@ func (app *SubresourceAPIApp) getVirtHandlerConnForVMI(vmi *v1.VirtualMachineIns
 }
 
 func getChangeRequestJson(vm *v1.VirtualMachine, changes ...v1.VirtualMachineStateChangeRequest) (string, error) {
```

Comment:
Would it make sense to return directly byte here instead of string?

Reply:
Done! thanks